### PR TITLE
added UI test to check all fragments of MainActivity

### PR DESCRIPTION
**Pull request by viveksb007**
Partially Fixes issue #333 

Changes:
- Added Test to open all fragments of MainActivity.

Checked by ```./gradlew connectedAndroidTest``` works fine.

Screenshots for the change: 
NA

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -65,4 +65,4 @@ after_success:
   - 'if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then ./upload-apk.sh; fi'
 
 notifications:
-  slack:
+  slack: fossasia:bqOzo4C9y6oI6dTF8kO8zdxp
```
Comment (CloudyPadmal) [inline .travis.yml]: Are we using Slack?

Comment (viveksb007) [inline .travis.yml]: Yeah we have a ```Pocket Science``` channel on slack.

File: app/src/androidTest/java/org/fossasia/pslab/MainActivityTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    @Rule
+    public ActivityTestRule<MainActivity> mActivityRule = new ActivityTestRule<MainActivity>(MainActivity.class);
+
+    @Test
+    public void checkingAllFragments() throws InterruptedException {
+        Thread.sleep(2000);
+
+        onView(withId(R.id.drawer_layout)).perform(DrawerActions.open());
+        onView(withId(R.id.nav_view)).perform(NavigationViewActions.navigateTo(R.id.nav_applications));
+        Thread.sleep(1000);
```
Comment (CloudyPadmal) [inline app/src/androidTest/java/org/fossasia/pslab/MainActivityTest.java]: Are these sleep events for the time to open up fragments right? :)

File: .travis.yml (hunk 1 above)
Comment (CloudyPadmal) [inline .travis.yml]: Is that still in use?

File: app/src/androidTest/java/org/fossasia/pslab/MainActivityTest.java (hunk 2 above)
Comment (viveksb007) [inline app/src/androidTest/java/org/fossasia/pslab/MainActivityTest.java]: yes Instrumentation tests are connected test i.e they are performed on an emulator or an actual device. So if we won't provide the delay, there is a possibility that the intended view didn't came in foreground yet.

File: .travis.yml (hunk 1 above)
Comment (viveksb007) [inline .travis.yml]: No one interacts there anymore. Its filled with pslab repos build status only.

**Pull request merged**